### PR TITLE
Adding LinkedIn Icon #27

### PR DESCRIPTION
**Pull request by adambkaplan**

### CODE REVIEW

File: layouts/partials/social-follow.html [hunk 1]
```diff
@@ -16,12 +16,17 @@
...
 {{ end }}
 {{ with .Site.Params.youtube }}
-  <a href="{{ . }}" class="link-transition youtube link dib z-999 pt3 pt0-l mr3" title="Youtube link">
+  <a href="{{ . }}" class="link-transition youtube link dib z-999 pt3 pt0-l mr2" title="Youtube link">
     {{ partial "svg/youtube.svg" (dict "size" $icon_size) }}
   </a>
 {{ end }}
 {{ with .Site.Params.github }}
-  <a href="{{ . }}" class="link-transition github link dib z-999 pt3 pt0-l mr3" title="Github link">
+  <a href="{{ . }}" class="link-transition github link dib z-999 pt3 pt0-l mr2" title="Github link">
```
Comment (adambkaplan) [inline layouts/partials/social-follow.html]: Fixed spacing of icons

File: src/css/_social-icons.css [hunk 2]
```diff
@@ -20,3 +20,7 @@
 .github:hover {
   fill: #FFFFFF;
 }
+
+.linkedin:hover {
+  fill: #0277B5
+}
```
Comment (adambkaplan) [inline src/css/_social-icons.css]: Hover on LinkedIn uses brand color.

File: layouts/partials/social-follow.html (hunk 1 above)
Comment (budparr) [reply layouts/partials/social-follow.html]: We should actually make it so the last one has mr3 and the others have mr2, but this is okay.

**Comment by adambkaplan** [comment]
This work duplicates #51 as an FYI...

**Comment by budparr** [comment]
Hey, @adambkaplan Thanks for this. I pulled in the first PR. Seems like the difference is the padding. I think I have a problem with the YT icon and need to address them all. I'm doing that for a 2.0 release (where I'm going to better integrate them with [Hugo's internal template](https://github.com/gohugoio/hugo/blob/0d29a0f7819e8d73149701052c29f090cd6db42b/tpl/tplimpl/template_embedded.go#L223), but if you want to tackle it before hand in a separate PR, it's fine with me.

**Comment by budparr** [comment]
Thanks @adambkaplan and @bdashrad I fixed the spacing issue in master.

**Pull request closed without merging**